src/extensions/qdatabase.c

```diff
@@ -176,6 +193,12 @@ qdb_t *qdb(const char *dbtype, const char *addr, int port, const char *username,
         return NULL;
     }
 
+#ifdef Q_ENABLE_PGSQL
+    if (autocommit == false) {
+        return NULL;
+    }
+#endif
```

Comment:
What's the reason we're not supporting manual commits in pgsql?

Reply:
There are differences about transactions between mysql and pgsql , please see comments of new commits